Codecademy/gamut
# chore(Badge): update colors

### Overview


Updates Badges to new colors


### PR Checklist

- [x] Related to designs: [here](https://www.figma.com/file/ReGfRNillGABAj5SlITalN/%F0%9F%93%90-Gamut?type=design&node-id=29959-39721&mode=design&t=aiZ3RL3h0uLBGJ9Q-0)
- [x] Related to JIRA ticket: [web-2485]

#### Testing Instructions


1. Go to the Storybook preview + check out the new colors on secondary + tertiary variants 


#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | https://github.com/codecademy-engineering/Codecademy/pull/36683 | [Monolith Env](http://www.google.fr/ 'Named link title') |
| Mono       | https://github.com/codecademy-engineering/mono/pull/4058  | [Portal Env](http://www.google.fr/ 'Named link title')   |
